docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst

```diff
@@ -105,3 +105,9 @@ Aiven evaluation process for new Apache Kafka Connect connectors checks:
 
     When requesting connectors that are not on the pre-approved list through a support ticket, specify the target Aiven for Apache Kafka service you'd like to have it installed to.
 
+
+
+------
+
+*Elasticsearch is a trademark of Elasticsearch B.V., registered in the U.S. and in other countries.*
```

Comment:
Do we need to state something about MongoDB as well?

Reply:
I *think* I checked, but I shall do so again...

https://www.mongodb.com/legal/trademark-usage-guidelines doesn't say anything about requiring an annotation, so I think this is OK.